.github/workflows/build-metrics-server.yml

```
@@ -0,0 +1,110 @@
+            .
+
+      - name: Build and tag codex-metrics-grafana
+        run: |
+          docker build \
+            -t codex-metrics-grafana:${{ github.sha }} \
+            -t $AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/codex-metrics-grafana \
+            -f packages/metrics_server/Dockerfile.grafana \
+            packages/metrics_server
+
+      - name: Push images
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:37: Double quote to prevent globbing and word splitting

(shellcheck)